ephemeraldata.adoc

```diff
@@ -35,40 +35,85 @@ through the following process:
 1. Install `SSDB` and note the ip address and port configured since you will
    need this data for the next steps. Refer to
    <<_ssdb_configuration_options>> for more information.
-2. Configure the {product-name} to utilize `SSDB`.
-3. Migrate any existing short-lived data to `SSDB` using the `/opt/zimbra/bin/zmmigrateattrs` command.
+2. Migrate any existing short-lived data to `SSDB` using the `/opt/zimbra/bin/zmmigrateattrs` command.
+3. Configure the {product-name} to utilize `SSDB`.
```

Comment:
Ah, I did this because the top of the file says "There are 3 main types of ephemeral data stored in LDAP during normal operation of the {product-name}". I do agree it sounds weird though, so I'll change it.